test/unit/email_rate_limit_test.rb

```diff
@@ -12,10 +12,12 @@ def seed_array
     valid_values          = seed_array.map { |n| n * multiplier }
     invalid_values        = last_clicked_days_ago.to_a - valid_values
     day_ago               = rand(last_clicked_days_ago)
-    clicked_ago           = valid_values.sample
-    assert EmailRateLimit.new(day_ago).now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
+    clicked_ago           = valid_values.reject { |int| (int % 7).zero? }.sample
+    assert EmailRateLimit.new(day_ago).now?(clicked_ago),
+      "Expected  EmailRateLimit.new(#{day_ago}).now?(#{clicked_ago}) to be true, was not"
     # Use user email frequency settings
-    assert_not EmailRateLimit.new(day_ago, minimum_frequency: "once_a_week").now?(clicked_ago), "Expected  EmailRateLimit.new(#{day_ago}, minimum_frequency: 'once_a_week').now?(#{clicked_ago}) to be false, was not"
+    assert_not EmailRateLimit.new(day_ago, minimum_frequency: "once_a_week").now?(clicked_ago),
+      "Expected  EmailRateLimit.new(#{day_ago}, minimum_frequency: 'once_a_week').now?(#{clicked_ago}) to be false, was not"
```

Comment:
When a user sets their email settings to "once_a_week" then we want to make sure that they don't get an email for any frequency lower, which in this case would be daily.

It's testing that the logic provided by the `minimum_frequency:` option works.